sdk/core/Azure.Core/src/TokenCredential.cs

```diff
@@ -18,6 +18,7 @@ public abstract class TokenCredential
         /// <param name="requestContext">The <see cref="TokenRequestContext"/> with authentication information.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> to use.</param>
         /// <returns>A valid <see cref="AccessToken"/>.</returns>
+        /// <remarks>Caching of the <see cref="AccessToken"/> returned is considered the responsibility of the caller; each call will request a fresh token being requested.</remarks>
```

Comment:
Nit:
```suggestion
        /// <remarks>Caching of the <see cref="AccessToken"/> returned is considered the responsibility of the caller: each call should request a fresh token being requested.</remarks>
```
I suggest "should" because this is an abstraction and you're telling implementers what they should do. The colon is because you're clarifying the first clause.

Reply:
Good suggestion!   Updated both methods with this.